### PR TITLE
Adding access to page buttons on PrintPreviewDialog by keyboard shortcut keys

### LINKED ISSUE #3682: [Accessibility] Keyboard shortcut key is unavailable to access the buttons(One Page, Two Page...)
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20377.7

* Have you experienced this same bug with .NET Framework?: 
Yes

**More Info:**

1. This issue can also repro on Framework project.
2. There is a similar bugs: [1156785 ](https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1156785?src=WorkItemMention&src-action=artifact_link)in VSO.
3. We can just use ‘Enter’ key enter to buttons(One Page, Two Page...).

**Minimal repro:**
1. Extract “[WindowsFormsApp3.zip](https://github.com/dotnet/winforms/files/5005021/WindowsFormsApp3.zip)” and open it in VS.
2. Build and run the project.
3. Using Tab key to navigate to Preview button, then press Enter key to show Print preview dialog.
4. Check whether any shortcut key is provided to buttons(One Page, Two Page...) to access.

**Expected behavior:**
Keyboard shortcut key should be available to access the buttons(One Page, Two Page...).

**Actual behavior:**
Keyboard shortcut key is unavailable to access the buttons(One Page, Two Page...).

### PR DESCRIPTION
**Pull request by SergeySmirnov-Akvelon**
Fixes #3682

## Proposed changes

- Adding support of keyboard shortcut keys for access to page buttons on PrintPreviewDialog

## Customer Impact
- The user will be able to switch between viewing modes using the shortcut keys (Ctrl+1, Ctrl+2, etc) on PrintPreviewDialog

## Regression? 

-  No

## Risk

- Minimal

## Test methodology 

- Manual testing

## Test environment(s) 
- Microsoft Windows [Version 10.0.19041.388]
- NET Core 5.0.100-rc.1.20420.14

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/3929)

### CODE REVIEW

File: src/System.Windows.Forms/src/Resources/System/Windows/Forms/PrintPreviewDialog.resx [hunk 1]
```diff
@@ -165,46 +165,46 @@
...
   <data name="sixpagesToolStripButton.AccessibleDescription">
     <value xml:space="preserve">Display six pages</value>
   </data>
   <data name="sixpagesToolStripButton.AccessibleName">
-    <value xml:space="preserve">Six pages</value>
+    <value xml:space="preserve">Six pages (Ctrl+5)</value>
   </data>
   <data name="sixpagesToolStripButton.Text">
-    <value xml:space="preserve">Six pages</value>
+    <value xml:space="preserve">Six pages (Ctrl+5)</value>
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/Resources/System/Windows/Forms/PrintPreviewDialog.resx:207]: What was the reason to go with `Ctrl+5` and not with `Ctrl+6`? Did you find any guidance about it?

Comment (RussKie) [reply src/System.Windows.Forms/src/Resources/System/Windows/Forms/PrintPreviewDialog.resx:207]: This was my suggestion https://github.com/dotnet/winforms/issues/3682#issuecomment-689948410 that everyone has agreed with.

In my opinion it is a logical sequence - option 1, 2, ...5, and not based on number of pages. E.g. if there was a sixth option to show 12 pages - as a user I wouldn't expect <kbd>CTRL</kbd>+<kbd>12</kbd> but <kbd>CTRL</kbd>+<kbd>6</kbd>.

File: Winforms.sln [hunk 2]
```diff
@@ -283,14 +285,10 @@ Global
...
 		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Release|Any CPU.ActiveCfg = Release|x86
 		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Release|Any CPU.Build.0 = Release|x86
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Debug|x86.ActiveCfg = Debug|x86
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Debug|x86.Build.0 = Debug|x86
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Release|x86.ActiveCfg = Release|x86
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Release|x86.Build.0 = Release|x86
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Debug|x64.ActiveCfg = Debug|x64
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Debug|x64.Build.0 = Debug|x64
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Release|x64.ActiveCfg = Release|x64
-		{C272DA06-B98D-4BB7-B1C4-ECF58F54B224}.Release|x64.Build.0 = Release|x64
```
Comment (RussKie) [inline Winforms.sln:293]: Why have these been deleted?

File: src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewDialog.cs [hunk 3]
```diff
@@ -1272,6 +1272,27 @@ protected override bool ProcessDialogKey(Keys keyData)
                         return false;
                 }
             }
+            else if ((keyData & Keys.Control) == Keys.Control)
+            {
+                Func<PrintPreviewDialogToolStripButton, bool> performPageClick = (PrintPreviewDialogToolStripButton pageToolStripButton) =>
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewDialog.cs:1277]: It is better to use local functions vs expressions, as they have less overhead (http://mustoverride.com/local_functions/).
Convert to a local function and place it after line:1293

**Review by RussKie — COMMENTED** [review]
👍

File: Winforms.sln (hunk 2 above)
Comment (SergeySmirnov-Akvelon) [reply Winforms.sln]: Changes are reverted

**Review by RussKie — DISMISSED** [review]

File: Winforms.sln (hunk 2 above)
Comment (RussKie) [reply Winforms.sln:293]: ❗  This is System.Windows.Forms.Interop.Tests which is a C/C++ project that must specify x86 and x64 platforms explicitly.

**Review by RussKie — CHANGES_REQUESTED** [review]

**Review by RussKie — APPROVED** [review]

File: src/System.Windows.Forms/src/Resources/System/Windows/Forms/PrintPreviewDialog.resx (hunk 1 above)
Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/src/Resources/System/Windows/Forms/PrintPreviewDialog.resx:207]: Had you guys looked at what Office or windows are doing in similar cases?

**Pull request merged**